app-misc/digitemp/files/digitemp-3.7.2-conflict-types.patch

```diff
@@ -0,0 +1,32 @@
+From fa56b0f78d12f97ac44e0a367d413a9e88611d1c Mon Sep 17 00:00:00 2001
```

Comment:
Missing a link to where the patch is from & gentoo bugs at the top, but I'll allow it here. Just please always do it in future